gitgud/__main__.py

```diff
@@ -253,20 +258,31 @@ def handle_levels(self, args):
 
     def handle_load(self, args):
         self.assert_initialized(skip_level_check=True)
-        if args.skill_name in all_skills:
-            skill = all_skills[args.skill_name]
+        
+        argset = args.skill_name.split("-")
 
+        if (len(argset) == 2):
+            args.skill_name, args.level_name = tuple(argset)
+        elif(len(argset) > 2):
+            print("You put too many dashes! Use at most one dash when trying to load a level.")
```

Comment:
Too many dashes is a weird error, do you think you could find the index of the dash and then do
```
dash_index = whatever
args.skill_name = args.skill_name[:dash_index]
args.level_name = args.skill_name[dash_index]
``` 
should then throw an error like
```
$ git gud load basic-s-committing
No skill "basic"
```
```
$ git gud load basics-s-committing
No level "s-committing"
```

Reply:
The error handling is already implemented, you'd just be passing the bad information along